tests/core/test_infer.py

```diff
@@ -30,11 +30,13 @@ def __init__(self):
 
     @property
     def input_ports(self):
-        return {"mod_in": NeuralType({0: AxisType(BatchTag), 1: AxisType(BaseTag, dim=1)})}
+        # return {"mod_in": NeuralType({0: AxisType(BatchTag), 1: AxisType(BaseTag, dim=1)})}
+        return {"mod_in": NeuralType(ChannelType(), (AxisType(AxisKind.Batch), AxisType(AxisKind.Dimension, 1)))}
```

Comment:
why is this not using axis=('B', 'D') notation?

Reply:
because it's shorter :)

Reply:
shorter? I do not understand...

Still, there is:
"AxisType(AxisKind.Dimension, 1)"
Can you add "1" to dimension when you are using the *axes=("B","D")* definition of axes?

Reply:
"Can you add "1" to dimension when you are using the axes=("B","D") definition of axes?" - No, you will have to use verbose version for that

Reply:
also, added "axis=" in many places